tests/main/refresh-all-undo/task.yaml

```diff
@@ -23,6 +23,17 @@ prepare: |
     echo "And the daemon is configured to point to the fake store"
     setup_fake_store $BLOB_DIR
```

Comment:
@sergiocazzolato  I think it's because that original code was using "exit" and not "return" on the happy path inside the function, anyway I'm testing on a different PR and can repush here if that's true